Sources/Alchemy/SQL/QueryBuilder/Query.swift

```diff
@@ -241,33 +436,59 @@ public class Query: Sequelizable {
         )
     }
 
+    /// Group returned data by a given column.
+    ///
+    /// - Parameters:
+    ///   - group: The table column to group data on
+    /// - Returns: The current query builder `Query` to chain future queries to
     public func groupBy(_ group: String) -> Self {
         self.groups.append(group)
         return self
     }
 
+    /// Order the data from the query based on given clause.
+    ///
+    /// - Parameters:
+    ///   - order: The `OrderClause` that defines the ordering
+    /// - Returns: The current query builder `Query` to chain future queries to
     public func orderBy(_ order: OrderClause) -> Self {
         self.orders.append(order)
         return self
     }
 
+    /// Order the data from the query based on a column and direction.
+    ///
+    /// - Parameters:
+    ///   - column: The column to order data by
+    ///   - direction: The `OrderClause.Sort` direction (either `asc` or `desc`)
+    /// - Returns: The current query builder `Query` to chain future queries to
     public func orderBy(column: Column, direction: OrderClause.Sort = .asc) -> Self {
-        //TODO: Add support for sort subquery
         return self.orderBy(OrderClause(column: column, direction: direction))
```

Comment:
nit; I've been omitting `return`s wherever possible

Reply:
Mmmm I feel like it hurts code legibility a bit, but for one liners prob fine